docroot/modules/custom/civictheme_migrate/src/Utils/CivicthemeTrait.php

```diff
@@ -0,0 +1,762 @@
+      static::civicthemeThemeLight(),
+      static::civicthemeThemeDark(),
+    ];
+  }
+
+  /**
+   * Spacing - none.
+   */
+  public static function civicthemeVerticalSpacingTypeNone() {
+    return 'none';
+  }
+
+  /**
+   * Spacing - top.
+   */
+  public static function civicthemeVerticalSpacingTypeTop() {
+    return 'top';
+  }
+
+  /**
+   * Spacing - bottom.
+   */
+  public static function civicthemeVerticalSpacingTypeBottom() {
+    return 'bottom';
+  }
+
+  /**
+   * Spacing - both.
+   */
+  public static function civicthemeVerticalSpacingTypeBoth() {
+    return 'both';
+  }
+
+  /**
+   * Types of spaces.
+   */
+  public static function civicthemeVerticalSpacingTypes() {
+    return [
+      static:: civicthemeVerticalSpacingTypeNone(),
+      static:: civicthemeVerticalSpacingTypeTop(),
+      static:: civicthemeVerticalSpacingTypeBottom(),
+      static:: civicthemeVerticalSpacingTypeBoth(),
+    ];
+  }
+
+  /**
+   * Banner type - Default.
+   */
+  public static function civicthemeBannerTypeDefault() {
+    return 'default';
+  }
+
+  /**
+   * Banner type - Large.
+   */
+  public static function civicthemeBannerTypeLarge() {
+    return 'large';
+  }
+
+  /**
+   * Types of banner.
+   */
+  public static function civicthemeBannerTypes() {
+    return [
+      static::civicthemeBannerTypeDefault(),
+      static::civicthemeBannerTypeLarge(),
+    ];
+  }
+
+  /**
+   * Image position - left.
+   */
+  public static function civicthemeImagePositionLeft() {
+    return 'left';
+  }
+
+  /**
+   * Image position - right.
+   */
+  public static function civicthemeImagePositionRight() {
+    return 'right';
+  }
+
+  /**
+   * Image blend-mode.
+   */
+  public static function civicthemeImageBlendMode() {
+    return 'soft-light';
+  }
+
+  /**
+   * Small size name.
+   */
+  public static function civicthemeSizeSmall() {
+    return 'small';
+  }
+
+  /**
+   * Large size name.
+   */
+  public static function civicthemeSizeLarge() {
+    return 'large';
+  }
+
+  /**
+   * Static Topics.
+   */
+  public static function civicthemeStaticTopics($count = NULL) {
+    return static::staticTerms('civictheme_topics', $count);
+  }
+
+  /**
+   * Static Topic.
+   */
+  public static function civicthemeStaticTopic() {
+    $entities = static::civicthemeStaticTopics(1);
+
+    return count($entities) > 0 ? reset($entities) : NULL;
+  }
+
+  /**
+   * Expose single listing filter type.
+   */
+  public static function civicThemeExposeSingleFilter() {
+    return ['topic'];
+  }
+
+  /**
+   * Expose multiple filter types.
+   */
+  public static function civicThemeExposeMultipleFilters() {
+    return ['topic', 'type', 'title'];
+  }
+
+  /**
+   * Static Site sections.
+   */
+  public static function civicthemeStaticSiteSections($count = NULL) {
+    return static::staticTerms('civictheme_site_sections', $count);
+  }
+
+  /**
+   * Static Site section.
+   */
+  public static function civicthemeStaticSiteSection() {
```

Comment:
please remove this and other methods that are used to generate static stubs - they are not supposed to be used for migrations